instrumentation-core/spring/spring-webflux-5.0/README.md

```diff
@@ -0,0 +1,87 @@
+# Manual Instrumentation for Spring-WebFlux
+
+Provides OpenTelemetry instrumentation for Spring's WebClient.
+
+## Quickstart
+
+### Add these dependencies to your project.
+
+Replace `SPRING_VERSION` with the version of spring you're using.
+`Minimum version: 5.0`
+
+Replace `OPENTELEMETRY_VERSION` with the latest stable [release](https://mvnrepository.com/artifact/io.opentelemetry).
+`Minimum version: 0.8.0`
+
+For Maven add to your `pom.xml`:
+```xml
+<dependencies>
+  <!-- opentelemetry -->
+  <dependency>
+    <groupId>io.opentelemetry.instrumentation</groupId>
+    <artifactId>opentelemetry-spring-webflux-5.0</artifactId>
+    <version>OPENTELEMETRY_VERSION</version>
+  </dependency>
+
+   <!-- provides opentelemetry-sdk -->
+   <dependency>
+    <groupId>io.opentelemetry</groupId>
+    <artifactId>opentelemetry-exporters-logging</artifactId>
+    <version>OPENTELEMETRY_VERSION</version>
+  </dependency>
+
+  <!-- required to instrument spring-webflux -->
+  <!-- this artifact should already be present in your application -->
+  <dependency>
+    <groupId>org.springframework</groupId>
+    <artifactId>spring-webflux</artifactId>
+    <version>SPRING_VERSION</version>
+  </dependency>
+
+</dependencies>
+```
+
+For Gradle add to your dependencies:
```

Comment:
i think copy same comments from pom above into gradle below, since gradle users will probably just skip the above section